lib/Service/MaintenanceService.php

```diff
@@ -217,7 +217,7 @@ private function lockMaintenanceRun(): void {
 	/**
 	 * every minute
 	 */
-	private function runMaintenance1(): void {
+	private function runMaintenance1(bool $forceRefresh = false): void {
```

Comment:
Is there a reason to pass `$forceRefresh` to all `runMaintenanceX()` functions even though it's only used in `runMaintenance5()`?